### ISSUE TITLE
Redundant public method

### ISSUE THREAD

**Issue by dmarcos-au** [archit737/code-challenge#2]
https://github.com/archit737/code-challenge/blob/3eaa897237c27d1ccd6aad272f20cbbf0a70be89/src/main/java/com/transurban/utils/cloud/CloudStorageInteract.java#L12

Wondering if we need to expose this in the interface as it's currently called within the `performReadAction` call.

**Comment by archit737** [comment]
Correct. But this was done on purpose. In this specific case it's being called within `performReadAction()` but might not work the same way for, say, One Drive. If we do decide to extend this to other cloud storage platform we'll need come up with a solution that works for every platform which might include exposing the `cloudServiceAuthenticate() `method
